frontend-react/src/components/MessageTracker/QualityFilters.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+import { Destination } from "../../resources/ActionDetailsResource";
+
+//TODO: do we want to create a new interface here or use the existing Destination?
```

Comment:
Even if we stick with the old Destination, if it's something that's coming back from the API, I vote for moving it into `src/config/endpoints/[some file that makes sense to put it in].ts`. It'll make the cleanup of those rest-hooks Resource classes a lot easier when we come around to it 😃